schedule.html

```diff
@@ -244,7 +244,7 @@ <h4 id="schedule-day-2">
           <tr>
             <td></td>
             <th>
-              08:00–09:00 <span>AM</span>
+              09:00–09:30 <span>AM</span>
```

Comment:
Would this work better https://github.com/w3c/vr-workshop/pull/43/commits/c07719f15c2020689ac96ca399b4a7c8594e28e3